{{cookiecutter.project_name}}/tests/test_token.py

```diff
@@ -174,7 +174,7 @@ def test_mint(token, owner, receiver):
     totalSupply = token.totalSupply()
     assert totalSupply == 1000
 
-    receiver_balance = token.balanceOf(receiver) 
+    receiver_balance = token.balanceOf(receiver)
```

Comment:
What are all these small changes doing? I can't see the difference

Reply:
There were extra white spaces at the end of those lines. You should be able to see the change in GitHub or in your text editor if you configure it to show every character.

![image](https://user-images.githubusercontent.com/21960643/188290334-178b6166-7427-48f0-a150-af640fa8465d.jpeg)

Reply:
okay, that's what I figured. it doesn't show up on mobile github